kano_settings/system/advanced.py

```diff
@@ -530,6 +554,7 @@ def set_parental_level(level_setting):
     else:
         set_chromium_parental('chromium' in enabled)
         set_dns_parental('dns' in enabled)
+        set_youtube_cookies('dns' in enabled)
```

Comment:
We should actually create a new feature string here (something like `'youtube-cookies'`) and add it to the [preferred setting level above](https://github.com/KanoComputing/kano-settings/blob/0359d0acde833ea8f5a52ab44fe9aed477ac3bf5/kano_settings/system/advanced.py#L533-L542), i.e. something like:

```python
     feature_levels = [
         # Low
         ['blacklist', 'youtube-cookies'],
         # Medium
         ['dns'],
         # High
         ['chromium'],
         # Ultimate
         ['ultimate']
     ]
```